site/src/_pages/docs/0_index.mdx

```diff
@@ -6,7 +6,7 @@ The Block Protocol is an open standard for building block-based interfaces. It e
 
 The following pages provide a practical introduction for developers who want to:
 
-- [develop blocks](/docs/developing-blocks) and publish them to the block registry
+- [develop blocks](/docs/developing-blocks) and publish them to the Hub
```

Comment:
Do you perhaps want to link to https://blockprotocol.org/hub?

Reply:
Done in https://github.com/blockprotocol/blockprotocol/commit/c2dd3314d99913ea5334b2d0e905cefa8df14c42